index.js

```diff
@@ -0,0 +1,82 @@
+import { DateTime } from 'luxon';
+import { Store } from './storage.js';
+import { UI } from './UI.js';
+import { displayList, displayForm, displayContact } from './display.js';
```

Comment:
- [ ] Great job installing the `Luxon` library, :+1:  but currently, the current data is not visible in the UI, I'll suggest you check your implementation and fix this as its part of the requirement for this milestone. :+1: :wave: 

- [ ] You did great breaking down your functions into files, But I think your code will be more organized and professional if you create a `module` folder in your application root and then move all the js files into the folder. :+1: :smile:  

- [ ] As part of the requirement, After refactoring your code, The functionality of adding and removing book should still work, but that's not the case here. I think you should go through the code one more time to get these features to be functional. :smile: :wave: :100: 

- - VIDEO FOR REFERENCE:

https://user-images.githubusercontent.com/107632780/236909019-d23828d3-fb8c-42f2-83af-45e8384b1672.mp4